cli/internal/fs/turbo_json.go

```diff
@@ -153,37 +160,20 @@ func (to TaskOutputs) Sort() TaskOutputs {
 	return TaskOutputs{Inclusions: inclusions, Exclusions: exclusions}
 }
 
-// ReadTurboConfig toggles between reading from package.json or the configFile to support early adopters.
-func ReadTurboConfig(rootPath turbopath.AbsoluteSystemPath, rootPackageJSON *PackageJSON) (*TurboJSON, error) {
-
-	turboJSONPath := rootPath.UntypedJoin(configFile)
-
-	// Check if turbo key in package.json exists
-	hasLegacyConfig := rootPackageJSON.LegacyTurboConfig != nil
-
+// ReadTurboConfig reads turbo.json from a provided path
+func ReadTurboConfig(turboJSONPath turbopath.AbsoluteSystemPath) (*TurboJSON, error) {
 	// If the configFile exists, use that
 	if turboJSONPath.FileExists() {
 		turboJSON, err := readTurboJSON(turboJSONPath)
 		if err != nil {
 			return nil, fmt.Errorf("%s: %w", configFile, err)
 		}
 
-		// If pkg.Turbo exists, log a warning and delete it from the representation
-		if hasLegacyConfig {
-			log.Printf("[WARNING] Ignoring \"turbo\" key in package.json, using %s instead.", configFile)
-			rootPackageJSON.LegacyTurboConfig = nil
-		}
-
 		return turboJSON, nil
 	}
 
-	// If the configFile doesn't exist, but a legacy config does, log that it's deprecated and return an error
-	if hasLegacyConfig {
-		log.Printf("[DEPRECATED] \"turbo\" in package.json is deprecated. Migrate to %s by running \"npx @turbo/codemod create-turbo-config\"\n", configFile)
-	}
-
-	// If there's no turbo.json and no turbo key in package.json, return an error.
-	return nil, errors.Wrapf(os.ErrNotExist, "Could not find %s. Follow directions at https://turbo.build/repo/docs to create one", configFile)
+	// If there's no turbo.json, return an error.
+	return nil, errors.Wrapf(os.ErrNotExist, "Could not find %s", configFile)
```

Comment:
Was there a reason to drop the link to the docs? It's causing the unit tests to fail.

Reply:
The goal was to make it more general purpose since this method will be used for turbo.json's other than the root one. But let me double check we didn't lose the error message for the existing use case.

Reply:
Great catch, it's back now.